### PR TITLE
[TTAHUB-1729] Smart Sheet events import CSV

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

Created a script to import Smart Sheet events. This populates the ownerId, regionId, and data blob columns. If a event already exists with the event id OR a creator cannot be found the event is skipped.

**Note:** Because we don't yet have Smart Sheet API access. This is just the first step of importing a CSV file from S3.

**cmd run:**
_yarn importSmartSheetEvent -f FILE_KEY_

## How to test

1. Review the code
2. Run the unit test
3. Import the CSV to MinIO from the JIRA 1729 and make sure everything is created correctly in the DB.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1729

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

**Review by thewatermethod — COMMENTED** [review]
Works great, the CLI should exit when complete though

**Review by nvms — APPROVED** [review]
looks good

I see you updated the filepath in the `importSmartSheetEvent:local` script in `package.json`, but you probably still want to rename the `importSmartSheetEventCli.js` file itself to have a capital `CLI` suffix.

also, any idea why the diff for `package.json` isn't just a few lines, but shows every line being changed? perhaps the line endings were somehow changed from LF to CRLF?

edit: the line endings are really a non-issue, mostly asking out of curiosity

**Comment by thewatermethod** [comment]
I believe to rename the file you have to run ```git mv [old] [new]```

File: docs/logical_data_model.puml
Comment (GarrettEHill) [inline docs/logical_data_model.puml:1]: Do you have any db changes not included? Why is this change coming through?

Comment (AdamAdHocTeam) [reply docs/logical_data_model.puml:1]: This goes with Jon's change for the new table structure.

Comment (AdamAdHocTeam) [reply docs/logical_data_model.puml:1]: I see what you are saying I merged main and the puml file changes went away.

**Review by thewatermethod — APPROVED** [review]

**Pull request merged**